actors/market/tests/market_actor_test.rs

```diff
@@ -729,7 +729,8 @@ fn provider_and_client_addresses_are_resolved_before_persisting_state_and_sent_t
     let end_epoch = start_epoch + 200 * EPOCHS_IN_DAY;
     rt.set_epoch(start_epoch);
 
-    let mut deal = generate_deal_proposal(client_bls, provider_bls, start_epoch, end_epoch);
+    let mut deal =
+        generate_deal_proposal(client_resolved, provider_resolved, start_epoch, end_epoch);
```

Comment:
Undo all changes to this file. The point of this test is to use the bls addresses to make sure the actor resolves things correctly.  Your code change doesn't break this test.